package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "bus-dashboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon -e js,json,mustache index.js",
+    "test": "jest --coverage --runInBand",
+    "tdd": "jest --watch --runInBand"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
+    "mongoose": "^5.9.21",
+    "mustache-express": "^1.3.0"
```

Comment:
I don't think we actually need mustache for this project. Let's leave it out until we do.

Reply:
Fixed.